ts/Dashboards/Components/KPIComponent/KPIComponent.ts

```diff
@@ -405,6 +432,37 @@ class KPIComponent extends Component {
         this.chart?.destroy();
         super.destroy();
     }
+
+    /**
+     * Gets a proper value, according to the provided formula option.
+     *
+     * @returns
+     * The formula value. Can be a number internally, or a string from the
+     * callback function.
+     *
+     * @internal
+     */
+    private getFormulaValue(): string|number|undefined {
+        const formula = this.options.formula;
+        const connector = this.getFirstConnector();
+        const table = connector?.table.modified;
+        const column = table?.getColumn(this.options.columnName);
+
+        if (!column || !formula) {
+            return;
+        }
+
+        if (isFunction(formula)) {
+            return formula.call(this, column);
+        }
+
+        try {
+            return KPIComponent.formulaFunctions[formula](column as Arguments);
+        } catch {
+            console.warn('Invalid formula option provided.'); // eslint-disable-line no-console
```

Comment:
It might not be related to this PR but if a wrong formula is declared, two warnings are displayed. 
Quickly investigate and create a separate issue if needed.

Reply:
Discussed internally, the problem is that the `getValue` method is/or may be used in different places, and the console log displays for each execution. It may be a wanted behaviour as the input data (or any other) may changed since the last iteration. It persists the same in this case but on the other hand, moving it somewhere to the top of the tree structure, may not be a suitable approach as well. Leaving it as it is.